cypress/component/DataSubmissionStudyInformation/ds_study_information.spec.js

```diff
@@ -0,0 +1,59 @@
+/* eslint-disable no-undef */
+import React from 'react';
+import { mount } from 'cypress/react';
+import { cloneDeep } from 'lodash/fp';
+import DataSubmissionStudyInformation from "../../../src/components/data_submission/ds_study_information";
```

Comment:
Codacy has a fix for the issue: [Strings must use singlequote.](https://app.codacy.com/gh/DataBiosphere/duos-ui/pullRequest?prid=9974908)
```suggestion
import DataSubmissionStudyInformation from '../../../src/components/data_submission/ds_study_information';
```